graphdatascience/tests/unit/conftest.py

```diff
@@ -12,7 +12,7 @@
 from graphdatascience.server_version.server_version import ServerVersion
 
 # Should mirror the latest GDS server version under development.
-DEFAULT_SERVER_VERSION = ServerVersion(2, 4, 0)
+DEFAULT_SERVER_VERSION = ServerVersion(2, 5, 0)
```

Comment:
Ah, that's a good one. We should add a checklist item for updating it on our branching card.